mpisppy/cylinders/spoke.py

```diff
@@ -187,22 +174,20 @@ def bound(self):
     def bound(self, value):
         self._append_trace(value)
         self._bound[0] = value
-        self.spoke_to_hub(self._bound)
+        self.spoke_to_hub(self._bound, self.bound_type())
+        return
 
     @property
     def hub_inner_bound(self):
         """Returns the local copy of the inner bound from the hub"""
-        return self._locals[-2]
+        # NOTE: This should be the same as _hub_bounds[1]
+        return self._hub_bounds[-2]
```

Comment:
Maybe should just make this `_hub_bounds[1]` then, since the code supports this more straightforward indexing now